HPCC-16355 Fix compile error for clienttools build

wangkx opened the pull request:
In HPCC-14879, zcrypt is added to thorhelper which is needed by
clienttools. So, the "PLATFORM" should be removed from:
HPCC_ADD_SUBDIRECTORY (zcrypt "PLATFORM")

Signed-off-by: wangkx kevin.wang@lexisnexis.com

wangkx committed:
HPCC-16355 Fix windows compile error for clienttools build

In HPCC-14879, zcrypt is added to thorhelper which is needed for
clienttools. So, the "PLATFORM" should be removed from:
HPCC_ADD_SUBDIRECTORY (zcrypt "PLATFORM")

Signed-off-by: wangkx <kevin.wang@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest 
Sha: 1e39f1838410366833273a2806c0238b5b059777
Build: success
Install hpccsystems-platform-community_6.1.0-trunk0.el6.x86_64.rpm
HPCC Start: OK
Unittest result: total:83 passed:83 failed:0

HPCC Stop: OK
HPCC Uninstall: OK

wangkx commented:
@afishbeck please review.

wangkx committed:
HPCC-16355 Fix compile error in dalift

In HPCC-16309, the system/security/shared is added into the include_directories
section of the dali/ft/ftslave.cmake. But, the ftslave.cmake is not used by
clienttools build which causes a compile error in dalift for the clienttools
build: Cannot open include file: 'seclib.hpp': No such file or directory. In
this fix, the system/security/shared is added into the include_directories
section of the dali/ft/dalift.cmake.

Signed-off-by: wangkx <kevin.wang@lexisnexis.com>

section of the dali/ft/ommit message for your changes. Lines starting

wangkx commented:
@RussWhitehead please review the last commit: https://github.com/hpcc-systems/HPCC-Platform/pull/9163/commits/880f652b5f2d060fc16f21bfad4822b3b25dfac9

HPCCSmoketest commented:
Automated Smoketest 
Sha: 880f652b5f2d060fc16f21bfad4822b3b25dfac9
Build: success
Install hpccsystems-platform-community_6.1.0-trunk0.el6.x86_64.rpm
HPCC Start: OK
Unittest result: total:83 passed:83 failed:0

HPCC Stop: OK
HPCC Uninstall: OK

RussWhitehead commented:
Looks good
